Arduino/victoria_teensy_ros_node/victoria_teensy_ros_node.ino

```diff
@@ -462,15 +444,18 @@ void doBlink() {
 void doBumperDebug() {
   // Read the bumper sensors
   snprintf(debug_str, sizeof(debug_str), 
-    "L: %d    R: %d    STOP: %d", bumper_left, bumper_right, bumper_stop);
+    "L: %d    R: %d", 
```

Comment:
Looks like we're going to have to have a style discussion in the future.
I prefer if parameters have to go multi-line, that it should be one parameter per line and aligned with the first parameter. Some say not.